autosklearn/automl.py

```diff
@@ -1040,6 +1045,8 @@ def fit(
     def fit_ensemble(self, y, task=None, precision=32,
                      dataset_name=None, ensemble_nbest=None,
                      ensemble_size=None):
+        # Make sure that input is valid
+        y = self.InputValidator.validate_target(y)
```

Comment:
could you please  check whether there's any overlap in functionality with _process_target_classes ?

Reply:
Yeah, I did the relevant changes to remove this method. Also dimensionality change after fit is included